staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -335,6 +338,11 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 					klog.V(2).Infof("%s: watch of %v returned 429 - backing off", r.name, r.expectedTypeName)
 					<-r.initConnBackoffManager.Backoff().C()
 					continue
+				case apierrors.IsInternalError(err):
+					if retryInternalError.Retry() {
```

Comment:
I would switch this to

```
case apierrors.IsInternalError(err) & retry.Retry():
  // We need to log something for sure - to know we are retrying
  continue
```

to avoid fallthrough (which is error-prone in future extensions)

Reply:
Done.